### DISCUSSION TITLE
Can't vote for Uniswap proposal

### DISCUSSION THREAD

**Discussion by smil3y7** [snapshot-labs/snapshot#957] [Q&A] [answered]
I'm trying to vote for active Uniswap proposal but even though my wallet is connected and I've had UNI tokens in it for several months it keeps showing my voting power is 0. What am I missing?

Thanks!!

**Comment by bonustrack** [comment, marked as answer]
On Uniswap space only delegated UNI may be used to vote on proposals. You can delegate to yourself or another address here: https://app.uniswap.org/#/vote

**Comment by smil3y7** [comment]
Thanks very much! Will give it a try once gas drops a little bit.
